### PR TITLE
Add Svelte and SvelteKit

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/johnmichel/Library-Detector-for-Chrome/issues/206
closes https://github.com/johnmichel/Library-Detector-for-Chrome/issues/207

### CODE REVIEW

File: library/libraries.js [hunk 1]
```diff
@@ -1971,5 +1971,39 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
...
+    },
+    'Svelte': {
+      id: 'svelte',
+      icon: 'svelte',
+      url: 'https://svelte.dev/',
+      npm: 'svelte',
+      test: function (win) {
+        // can be toggled with discloseVersion option
+        // see https://svelte.dev/docs/svelte-compiler#types-compileoptions
+        const versions = win.__svelte.v;
```
Comment (Conduitry) [inline library/libraries.js:1983]: @benmccann Do we need some try-catches or optional chaining here? `win.__svelte.v` is going to throw if someone's disabled `discloseVersion`, and we'll never get to the `document.querySelector('[class*="svelte-"]')` below.

**Comment by benmccann** [comment]
@johnmichel @housseindjirdeh would you be able to take a look at this one? thanks!

**Review by housseindjirdeh — APPROVED** [review]
Sorry for the delay, and thanks for submitting this!

LGTM

**Comment by benmccann** [comment]
Thanks so much @housseindjirdeh! Could I bother you for a new release as well?

**Pull request merged**